modules/is_analytics_worker_master/manifests/params.pp

```diff
@@ -0,0 +1,89 @@
+# ----------------------------------------------------------------------------
+#  Copyright (c) 2019 WSO2, Inc. http://www.wso2.org
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ----------------------------------------------------------------------------
+
+# Claas is_analytics_worker::params
+# This class includes all the necessary parameters.
+class is_analytics_worker_master::params {
+  $user = 'wso2carbon'
+  $user_group = 'wso2'
+  $product = 'wso2is-analytics'
+  $product_version = '5.7.0'
+  $profile = 'worker'
+  $service_name = "${product}-${profile}"
+
+  # Define the template
+  $template_list = [
+    'conf/worker/deployment.yaml'
+  ]
+
+  # -------------- Deploymeny.yaml Config -------------- #
```

Comment:
Spellings mistake `Deploymeny`

Reply:
Resolved https://github.com/wso2/puppet-is/pull/62/commits/d115f04492787980146f632a774a7a6d1f4f5466